src/actions/dataset.ts

```diff
@@ -13,6 +13,15 @@ import {SHELF_CLEAR} from './shelf';
 
 import {getConfig} from '../selectors';
 
+import {ExpandedType} from 'compassql/build/src/query/expandedType';
+
+export type FieldAction = DatasetSchemaChangeFieldType;
```

Comment:
This is changing dataset, so we don't really have to declare new group of actions.